sensor/tests/resource/helper.go

```diff
@@ -54,6 +54,18 @@ type YamlTestFile struct {
 	File string
 }
 
+// awaitedResources slice of resources that need to be awaited
+var awaitedResources = []string{"Service"}
+
+func isAwaitedResource(kind string) bool {
```

Comment:
nit: Just like the comment above, I think this one can be renamed to `shouldWaitForResource(...)`. It reads better in the code.

Reply:
Same as above